library/src/main/scala/scala/scalajs/js/BigInt.scala

```diff
@@ -0,0 +1,188 @@
+   * @return The wrapped primitive value of a BigInt object.
+   */
+  override def valueOf(): BigInt = js.native
+}
+
+/**
+ * An object with some or all of the following properties.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toLocaleString#Parameters
+ */
+trait BigIntOptions extends Object {
+  var localeMatcher: js.UndefOr[String] = js.undefined
+  var style: js.UndefOr[String] = js.undefined
+  var currency: js.UndefOr[String] = js.undefined
+  var currencyDisplay: js.UndefOr[String] = js.undefined
+  var useGrouping: js.UndefOr[Boolean] = js.undefined
+  var minimumIntegerDigits: js.UndefOr[Int] = js.undefined
+  var minimumFractionDigits: js.UndefOr[Int] = js.undefined
+  var maximumFractionDigits: js.UndefOr[Int] = js.undefined
+  var minimumSignificantDigits: js.UndefOr[Int] = js.undefined
+  var maximumSignificantDigits: js.UndefOr[Int] = js.undefined
+}
+
+object BigIntOptions {
+  def apply(
+    localeMatcher: js.UndefOr[String] = js.undefined,
+    style: js.UndefOr[String] = js.undefined,
+    currency: js.UndefOr[String] = js.undefined,
+    currencyDisplay: js.UndefOr[String] = js.undefined,
+    useGrouping: js.UndefOr[Boolean] = js.undefined,
+    minimumIntegerDigits: js.UndefOr[Int] = js.undefined,
+    minimumFractionDigits: js.UndefOr[Int] = js.undefined,
+    maximumFractionDigits: js.UndefOr[Int] = js.undefined,
+    minimumSignificantDigits: js.UndefOr[Int] = js.undefined,
+    maximumSignificantDigits: js.UndefOr[Int] = js.undefined
+  ): BigIntOptions = {
+    val obj = js.Dynamic.literal()
+    localeMatcher.foreach(_v => obj.updateDynamic("localeMatcher")(_v))
+    style.foreach(_v => obj.updateDynamic("style")(_v))
+    currency.foreach(_v => obj.updateDynamic("currency")(_v))
+    currencyDisplay.foreach(_v => obj.updateDynamic("currencyDisplay")(_v))
+    useGrouping.foreach(_v => obj.updateDynamic("useGrouping")(_v))
+    minimumIntegerDigits.foreach(_v => obj.updateDynamic("minimumIntegerDigits")(_v))
+    minimumFractionDigits.foreach(_v => obj.updateDynamic("minimumFractionDigits")(_v))
+    maximumFractionDigits.foreach(_v => obj.updateDynamic("maximumFractionDigits")(_v))
+    minimumSignificantDigits.foreach(_v => obj.updateDynamic("minimumSignificantDigits")(_v))
+    maximumSignificantDigits.foreach(_v => obj.updateDynamic("maximumSignificantDigits")(_v))
+    obj.asInstanceOf[BigIntOptions]
+  }
+}
+
+@js.native
+@JSGlobal
+object BigInt extends js.Object {
+  /**
+   * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+   */
+  def apply(value: Int): BigInt = js.native
+
+  /**
+   * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+   */
+  def apply(value: String): BigInt = js.native
+
+  /**
+   * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+   */
+  def apply(value: BigInt): BigInt = js.native
```

Comment:
Even though this works, it is silly to call this overload as it is basically an identity. I would not expose it.